Datastore/src/Query/AggregationQueryResult.php

```diff
@@ -93,6 +101,8 @@ public function __construct($result = [])
         if (isset($result['batch']['readTime'])) {
             $this->readTime = $result['batch']['readTime'];
         }
+
+        $this->mapper = $mapper ?? new EntityMapper('foo', true, false);
```

Comment:
Define in comment why default entity mapper is instantiated with random `foo` project id value.

Reply:
I've added the proper explaination now. PTAL @vishwarajanand